pkg/task/tasklastrun_test.go

```diff
@@ -82,6 +82,19 @@ func TestTaskrunLatest_two_run(t *testing.T) {
 				cb.TaskRunCompletionTime(secondRunCompleted),
 			),
 		),
+		tb.TaskRun("tr-3", tb.TaskRunNamespace("ns"),
+			cb.TaskRunCreationTime(secondRunCompleted),
+			tb.TaskRunLabel("tekton.dev/clusterTask", "task"),
```

Comment:
probably we can add one more label for `tekton.dev/task: "task` so that we have one case for backward compatibility

Reply:
done, thanks @vinamra28